decidim-elections/db/migrate/20201028090110_add_votes_to_decidim_elections_answers.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddVotesToDecidimElectionsAnswers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :decidim_elections_answers, :votes, :integer, default: 0, null: false
```

Comment:
As the `:votes` attribute is a sum of the total votes for the answer, and not a relation, I'd rename it to `:total_votes`

Reply:
I get the thought. I'm just not sure if `total_votes` might be confusing as these aren't the `total_votes` for an `election`. Do you know what I mean?

Reply:
maybe `votes_count` could be a better alternative?

Reply:
@slickepinne yes, but it's relative to the record and both related, it's easy to get the difference between them:

```
election.total_votes = answers.sum(:total_votes)
```

@leio's proposal is also good!